plugin/core/sessions.py

```diff
@@ -271,7 +271,7 @@ def get_initialize_params(variables: Dict[str, str], workspace_folders: List[Wor
         "rootPath": first_folder.path if first_folder else None,
         "workspaceFolders": [folder.to_lsp() for folder in workspace_folders] if workspace_folders else None,
         "capabilities": capabilities,
-        "initializationOptions": config.init_options.create_resolved(variables)
+        "initializationOptions": config.init_options.create_resolved(variables).get()
```

Comment:
Now that we *always* resolve these DottedDicts after possible plugin intervention, maybe we can call the method `resolve` and it'll return a `Dict[str, Any]` ?

Reply:
Would it resolve a copy of the Dict or the internal dict (`self._d`). Because current implementation actually resolves internal dict and I don't think that was intended.

Reply:
Changed create_resolved => get_resolved because I think `resolve` is a poor name due to being hard to search for and also sounding like it's gonna change the internal state.

Reply:
Oh, I think `sublime.expand_variables` will modify the passed-in object but it doesn't so I don't have to make a copy and the existing behavior was already correct.